update-sumbodules.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# Update tree-sitter submodules
+#
+# Usage: ./update-sumbodules.sh $tree-sitter-language
+
+# Update submodule
+git submodule update --remote $1
+
+# Recreate the language
+cd enums
```

Comment:
Nit: use `pushd enums` instead of this, and `popd` instead of `cd ..`, so if the command fails you are not placed in `enums` and have to do `cd ..` manually

Reply:
We need to set `bash` not `sh` though, because `pushd` and `popd` are bash built-in

Reply:
It's a "nit", so feel free to either apply the change or not